Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -45,7 +45,7 @@ namespace {
 ByteString s_main_program_path;
 
```

Comment:
I still don't understand why we need this.
Most libcs I know of don't go through all this to support something basically no one relies on.